locale/fi/LC_MESSAGES/django.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-09-27 12:10+0300\n"
```

Comment:
Please try to refrain from changing the timestamp on the locale files -- this will usually lead to merge conflicts.

Reply:
Resolved in eb4340f.